.github/workflows/images.yaml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   build-and-push:
     name: Build and push all images
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Doesn't that mean it will update automatically and may break CI?

Reply:
you are right, we have a FORCE_BUILD for this purpose, let me revert it back.